FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheckAvailability.h

```diff
@@ -16,17 +16,34 @@
 
 // Availability conditions for different App Check SDK components.
 
+#import <Foundation/Foundation.h>
```

Comment:
I struggled on this for awhile before realizing the Foundation import was needed to evaluate `defined(__WATCHOS_9_0)` as truthy. I don't know why because it **should** be in the prelude but the `Foundation` import definitely makes a difference... sigh